.github/workflows/ci.yml

```diff
@@ -77,20 +79,23 @@ jobs:
         python-version: ${{ matrix.python }}
         cache: pip
 
-    - name: Install nox and uv
+    - name: Install graphviz and pandoc for documentation builds
+      if: startsWith(matrix.name, 'Documentation')
+      run: sudo apt-get install graphviz pandoc
```

Comment:
This command only works for Ubuntu, so we would have to change this later on if we want to build docs on Windows or macOS in CI.  (It would be nice to have that capability, but I'm less worried about building docs across platforms than I am about running tests.)